src/chi-vue/src/components/data-table-bulk-actions/DataTableBulkActions.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import { findComponent } from '@/utils/utils';
+import { BULK_ACTIONS_CLASSES, CHECKBOX_CLASSES, FORM_CLASSES, UTILITY_CLASSES } from '@/constants/classes';
+import DataTable from '../data-table/DataTable';
+
+@Component({})
+export default class DataTableBulkActions extends Vue {
+  @Prop() selectedRowCount!: number;
+
+  _selectedRows(e: any) {
```

Comment:
Any reason this is of type any?

Reply:
srcElement is not getting picked by the event if type is Event. So changed it to any.

Reply:
Do we need `e.srcElement.checked` basically to emit `chiShowSelectedOnly` with an attached boolean `true / false` ?
In that case, we could simply add another parameter to the method, like:
`_selectedRows(e: Event, selected: boolean) {`

Reply:
Also, for consistency with other event emission methods, I would also suggest renaming the method to `_emitSelectedRows() {}`